packages/client/src/runtime/RequestHandler.ts

```diff
@@ -81,9 +82,9 @@ export class RequestHandler {
     this.hooks = hooks
     this.dataloader = new DataLoader({
       batchLoader: (requests) => {
-        const info = getRequestInfo(requests[0])
+        const info = getRequestInfo({ ...requests[0], tracingConfig: client._tracingConfig })
```

Comment:
The tracing config should already be on the request object
```suggestion
        const info = getRequestInfo(requests[0])
```